### PR TITLE
feat: test special filters for ELL tab

### PR DESCRIPTION
**Pull request by alexmcdermid**

### CODE REVIEW

File: tests-e2e/cypress/e2e/sdc-collection/stepThreeFilters.cy.ts [hunk 1]
```diff
@@ -218,6 +218,55 @@ describe('SDC School Collection View', () => {
       });
     });
 
+    it('verifies special filters for ELL tab', () => {
+      cy.intercept(Cypress.env('interceptors').collection_students_pagination).as('pagination');
+      cy.visit('/');
+      cy.get(selectors.dashboard.dataCollectionsTile).click();
+      cy.get(selectors.dataCollectionsLanding.continue).contains('Continue').click();
+      cy.get('button[value="English Language Learning"]').click();
```
Comment (mightycox) [inline tests-e2e/cypress/e2e/sdc-collection/stepThreeFilters.cy.ts:226]: 'button[value="English Language Learning"]' can go in the selectors file

File: tests-e2e/cypress/e2e/sdc-collection/stepThreeFilters.cy.ts [hunk 2]
```diff
@@ -218,6 +218,55 @@ describe('SDC School Collection View', () => {
...
+      cy.get(selectors.ellComponent.tab).contains('Filters').click();
+      cy.get(selectors.activeFiltersDrawer.drawer).contains('1-5 years in ELL').click();
+      cy.get(selectors.activeFiltersDrawer.drawer).contains('Apply Filters').click();
+
+      cy.get(selectors.studentLevelData.detailsLoadingBar).should('exist');
+      cy.get(selectors.ellComponent.tab).find(selectors.studentLevelData.studentsFound).should('exist').contains(1);
+      cy.get(selectors.ellComponent.tab).find('tbody tr').each($cell => {
+        cy.wrap($cell).children().last().invoke('text').then((text) => {
+          expect(text).to.satisfy((value: string) => {
+            return value === '17-English Language Learning3';
```
Comment (mightycox) [inline tests-e2e/cypress/e2e/sdc-collection/stepThreeFilters.cy.ts:237]: We expect the text to have a 3 at the end of it?

Comment (alexmcdermid) [reply tests-e2e/cypress/e2e/sdc-collection/stepThreeFilters.cy.ts:237]: Yes

File: tests-e2e/cypress/e2e/sdc-collection/stepThreeFilters.cy.ts (hunk 1 above)
Comment (alexmcdermid) [reply tests-e2e/cypress/e2e/sdc-collection/stepThreeFilters.cy.ts:226]: Hmm this is the pattern used in the other step 3 tests - I can put it in the selectors

**Review by mightycox — APPROVED** [review]

**Pull request merged**